lib/google/serverless/exec/tasks.rb

```diff
@@ -280,9 +280,12 @@ def show_usage
               CLOUD_BUILD_GCS_LOG_DIR
                 GCS bucket name of the cloud build log when GAE_STRATEGY is "cloud_build".
                 (ex. "gs://BUCKET-NAME/FOLDER-NAME")
+              PRODUCT
+                The serverless product to use. If not specified, autodetects which product,
+                {APP_ENGINE} or {CLOUD_RUN} to use.
```

Comment:
Remember we have two levels of interfaces. At this level, we are using environment variables whose values are strings, not constants. The constants apply to the Exec class, but here we have to use strings.